pkg/chain/gen/tmp/cmd.go

```diff
@@ -0,0 +1,13 @@
+package cmd
```

Comment:
I am confused, should we commit anything in `tmp` directory?

Reply:
Yeah, we probably shouldn't. I'm not sure why we've got this file output by the code generator.